.changeset/early-socks-enjoy.md

```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-application-data-cube': minor
+'@finos/legend-data-cube': patch
+'@finos/legend-shared': patch
+---
+
+DataCube: Add CSV File as one of the data cube source
```

Comment:
Remember to properly split the changeset, this changelog message is not appropriate for `@finos/legend-shared`

```suggestion
```